Issue 2 view the entries - YoonHwan Cho

YoonHCho opened the pull request:
Issue 2 view the entries - YoonHwan Cho
![Project 1 Issue 2 Finished](https://user-images.githubusercontent.com/106636383/179872408-1e330168-a209-4ebe-891d-779f2c986983.gif)

YoonHCho committed:
Completed list4 of issue 2 view entries

YoonHCho committed:
defined a function with most of journal entry, need to uncomment

YoonHCho committed:
Completed Add an anchor to the entries

YoonHCho committed:
completed issue-2

codingcodymiller commented on css/styles.css:
```diff
@@ -94,3 +97,51 @@ textarea:focus {
+  padding-left: 0;
+}
+
+.hidden {
+  display: none;
+}
```
This already exists in `layout.css`

codingcodymiller commented on index.html:
```diff
@@ -22,13 +22,20 @@
       <div class="row">
-        <h2 class="column-full font">Code Journal</h2>
+        <div class="column-full font-white ">
+          <ul class="padding-l0">
+            <li class="font nav-inline">Code Journal</li>
+            <li class="nav-inline margin-l20"><a class="nav-nounderline-white entry-view" data-view="entry-form">Entries</a></li>
+
+          </ul>
```
This isn't a list, so it shouldn't be a `ul`

codingcodymiller commented on js/main.js:
```diff
@@ -21,4 +20,90 @@ function submitEntry(event) {
+var $getForm = document.querySelector('.get-form');
+viewEntries.addEventListener('click', hideFormShowEntries);
+
+function hideFormShowEntries(event) {
+  $getForm.className = 'hidden';
+  $getEntries.className = 'get-entries';
+  data.view = 'entries';
+}
+
+var $newEntry = document.querySelector('.new-entry');
+$newEntry.addEventListener('click', showFormHideEntries);
+
+function showFormHideEntries(event) {
+  $getForm.className = 'get-entries';
+  $getEntries.className = 'hidden';
+  data.view = 'entry-form';
+
+}
```
This is a very manual way of handling view swapping, I suggest you try to implement a system more similar to what you did in the `javascript-view-swapping` exercise that used a loop over all view containers.

YoonHCho committed:
still working on swapping views

YoonHCho committed:
still working on view swapping

YoonHCho committed:
completed issue 2

YoonHCho commented:
1. removed the CSS rule set class hidden in styles.css
2. took out the ul element and replaced with section
3. used the view swapping concept from the exercise
![Project 1 Issue 2 Finished](https://user-images.githubusercontent.com/106636383/180042795-578a0c01-a706-4793-9002-93e6810255b3.gif)

codingcodymiller commented on js/main.js:
```diff
@@ -65,45 +67,32 @@ function loadView(event) {
-    var $noEntriesMsg = document.createElement('p');
-    $noEntriesMsg.textContent = 'No entries have been recorded.';
-    $noDiv.appendChild($noEntriesMsg);
+var $body = document.querySelector('body');
+
+$body.addEventListener('click', toggleView);
```
Instead of putting this on the `body` element, I would apply it directly to each `a` element that is used for navigating to a different view

YoonHCho committed:
Completed issue 2 part 3

YoonHCho commented:
changed the new button as `a` tag  and used it to loop through a tag to view swap


![Project 1 Issue 2 Finished part3](https://user-images.githubusercontent.com/106636383/180071202-444eb3c4-1749-4cc4-9843-ef820bfca2ba.gif)
